tools/scenario-player/scenario_player/tasks/blockchain.py

```diff
@@ -103,12 +104,27 @@ def __init__(
                 'Not all required keys provided. Required: ' + ', '.join(required_keys),
             )
 
-        self.contract_name = config.get('contract_name', None)
-        self.event_name = config.get('event_name', None)
-        self.num_events = config.get('num_events', 0)
```

Comment:
`.get` would return None by default so changing this might break if at some point the default value of `num_events` is supposed to be 0

Reply:
In line 102 it's required that all keys are available

Reply:
If it's like that, then why not just use `config['xxx']`. Using `.get()` makes the reader think that the keys can also not be available.